AuroraEditor/Base/FileSystemClient/Model/FileItem/FileItem.swift

```diff
@@ -58,9 +58,8 @@ public extension FileSystemClient {
             source.resume()
             self.watcher = source
 
-            // reindex the current item, because the files in the item may have changed
+            // TODO: reindex the current item, because the files in the item may have changed
             // since the initial load on startup.
```

Comment:
did you remove `fileSystemClient?.reloadFromWatcher(sourceFileItem: self)` intentionally?

Reply:
It caused folders to collapse so I think he removed it for that reason